types/next-redux-saga/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for next-redux-saga 3.0
+// Project: https://github.com/bmealhouse/next-redux-saga
+// Definitions by: Leo Cavalcante <https://github.com/leocavalcante>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { ComponentType } from "react";
+
+declare function nextReduxSaga<P = any>(arg: { async: boolean }): ((BaseComponent: ComponentType<P>) => ComponentType<P>);
+declare function nextReduxSaga<P = any>(arg: ComponentType<P>): ComponentType<P>;
+
+export default nextReduxSaga;
```

Comment:
`require("next-redux-saga")` is itself a function, not an object with a `default` property. So this should be `export = nextReduxSaga;`.